crates/lib-dialects/test/fixtures/dialects/tsql/apply_clause.yml

```diff
@@ -0,0 +1,69 @@
+file:
+- statement:
+  - select_statement:
+    - select_clause:
+      - keyword: SELECT
+      - select_clause_element:
+        - column_reference:
+          - naked_identifier: OrderIDs
+          - dot: .
+          - naked_identifier: value
+      - comma: ','
+      - select_clause_element:
+        - column_reference:
+          - naked_identifier: Details
+          - dot: .
+          - naked_identifier: ProductName
+    - from_clause:
+      - keyword: FROM
+      - from_expression:
+        - from_expression_element:
+          - table_expression:
+            - table_reference:
+              - table_reference:
+                - tsql_variable: '@OrderIDs'
+          - alias_expression:
+            - keyword: AS
+            - naked_identifier: OrderIDs
+        - join_clause:
+          - keyword: OUTER
+          - keyword: APPLY
+          - from_expression_element:
+            - table_expression:
+              - bracketed:
+                - start_bracket: (
+                - select_statement:
+                  - select_clause:
+                    - keyword: SELECT
+                    - select_clause_element:
+                      - expression:
+                        - column_reference:
+                          - naked_identifier: ProductName
+                        - comparison_operator:
+                          - raw_comparison_operator: =
+                        - column_reference:
+                          - naked_identifier: p
+                  - unparsable:
```

Comment:
nit: unparsable but the CI should show that.

Reply:
All this CI issues should be fixed